src/lua/LuaSerializer.cpp

```diff
@@ -13,6 +13,13 @@
 #include "core/Log.h"
 #include "profiler/Profiler.h"
 
+// Well-known names of various serialization-related caches stored in the
+// Lua Registry
+static const char *NS_REFTABLE = "PiSerializerTableRefs";
```

Comment:
This is for consistency's sake - rather than manually writing or copy-pasting the string name every time we need to access the table, all references in code to this table use a single constant variable as an identifier. It reduces the possibility of mis-spelling the table name and allows it to participate in autocompletion. It can also be made `extern` if it needs to be referred to outside this file.